datalab/datalab_session/management/commands/reconcile_file_cache.py

```diff
@@ -0,0 +1,13 @@
+import sys
+from django.core.management.base import BaseCommand
+
+from datalab.datalab_session.utils.filecache import FileCache
+
+
+class Command(BaseCommand):
+    help = 'Reconciles the filecache with the files currently in the tmp dir of the system. Meant to run on pod creation'
+    
+    def handle(self, *args, **kwargs):
+        FileCache().reconcile_cache()
```

Comment:
This management command does a verification on the cache in redis vs. what is currently on the file system and gets them in sync. I needed this for local development with skaffold since each deploy would clear the temp dir but not clear redis since the dependencies were not restarted as well. I think this is probably nice to have in production to keep us from getting out of sync when the pod is moved and has a new volume.